### PR TITLE
fix(docs): minor grammatical and typographical fixes in files

### PR DESCRIPTION
**Pull request by tmsagarofficial**
Checklist:

- [ ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Check and let me know with any suggestions.

### CODE REVIEW

File: docs/courses-vscode-extension.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This details the maintenance guidelines for the [freeCodeCamp/courses-vscode-ext
 
 ## Publishing the Extension
 
-A GitHub Action automagically publishes the extension to the Visual Studio Marketplace, on the release of a new GitHub Release.
```
Comment (jeremylt) [inline docs/courses-vscode-extension.md:7]: Ok, `automagically` is a great word. We should keep it!

File: docs/how-to-open-a-pull-request.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ When you are working on regular bugs and features on our development branch `mai
 
 ### For Upcoming Curriculum and Features
 
-When you are working on features for our upcoming curriculum `next-*` branches, you have to do a cherry pick:
+When you are working on features for our upcoming curriculum `next-*` branches, you have to cherry pick:
```
Comment (jeremylt) [inline docs/how-to-open-a-pull-request.md:146]:
```suggestion
When you are working on features for our upcoming curriculum `next-*` branches, you have to do a `cherry-pick`:
```

File: docs/how-to-work-on-tutorials-that-use-coderoad.md [hunk 3]
```diff
@@ -119,11 +119,11 @@ If you have write access to the freeCodeCamp repo, have verified the `CHANGELOG`
 - Make a PR to `main` on the freeCodeCamp repo
 - If you are satisfied, merge it or leave it and ask for a review from someone
 - After the PR is merged, open the tutorial by following the instructions on the [rdb-alpha repo](https://github.com/freeCodeCamp/rdb-alpha) to make sure it's loading properly, and that you can get through a few steps
-- Finally, if any PRs for this version exists, close them
+- Finally, if any PRs for this version exist, close them
 
 ### How to Revert to a Previous Version
 
-- Create a new branch off the latest `main` with `git checkout -b revert/to-version-X.X.X`
+- Create a new branch of the latest `main` with `git checkout -b revert/to-version-X.X.X`
```
Comment (jeremylt) [inline docs/how-to-work-on-tutorials-that-use-coderoad.md:126]:
```suggestion
- Create a new branch off of the latest `main` with `git checkout -b revert/to-version-X.X.X`
```

File: docs/moderator-handbook.md [hunk 4]
```diff
@@ -350,7 +350,7 @@ http://suicide.org/international-suicide-hotlines.html
 
 ## A Note on Free Speech
 
-Sometimes people will defend something offensive or incendiary that they said as "free speech."
+Sometimes people will defend something offensive or incendiary that they say as "free speech."
```
Comment (jeremylt) [inline docs/moderator-handbook.md:353]:
```suggestion
Sometimes people will defend something offensive or incendiary that they said as "free speech."
```

File: docs/courses-vscode-extension.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ This details the maintenance guidelines for the [freeCodeCamp/courses-vscode-ext
...
+A GitHub Action automatically publishes the extension to the Visual Studio Marketplace, on the release of a new GitHub Release.
```
Comment (jeremylt) [inline docs/courses-vscode-extension.md:7]:
```suggestion
A GitHub Action automagically publishes the extension to the Visual Studio Marketplace, on the release of a new GitHub Release.
```

**Review by huyenltnguyen — COMMENTED** [review]
I left some comments, but please don't update the PR yet as we will need other reviewers to weigh in.

It looks like you were editing the files via GitHub UI so I'm adding a small note here. While it's possible to make modifications directly on GitHub, we recommend forking the repo and making the changes locally instead. Making code changes locally would allow you to catch coding issues sooner and be more confident with your PRs.

File: docs/authors-analytics-manual.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 # Authors Analytics Manual
 
-If you are an author with access to the publication's Google Analytics Property (News), you can use this guide to view your articles engagement and search for articles by publication language.
+If you are an author with access to the publication's Google Analytics Property (News), you can use this guide to view your article's engagement and search for articles by publication language.
```
[the remark this answers is not in the document]
Comment (jeremylt) [reply docs/authors-analytics-manual.md:3]: I think using as an adjective (your option 2) would be better here

File: docs/how-to-work-on-tutorials-that-use-coderoad.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jeremylt) [reply docs/how-to-work-on-tutorials-that-use-coderoad.md:126]: I think "off of" is the most correct in this case

**Review by tmsagarofficial — COMMENTED** [review]
Pls check now

**Comment by Universe-stack** [comment]
Hi @jeremylt, I'd like to help resolve this issue if it's needed. Thanks.

File: docs/reply-templates.md [hunk 7]
```diff
@@ -232,7 +232,7 @@ Happy contributing.
 ```md
 We typically do not assign issues. Instead, we accept the first pull request that comprehensively solves the issue.
 
-Issues labelled with `help wanted` or `first timers only` are open for contributions.
+Issues labeled with `help wanted` or `first timers only` are open for contributions.
```
[the remark this answers is not in the document]
Comment (jeremylt) [reply docs/reply-templates.md:235]: Does the codebase use American English or British English spellings? `Labeled` is American English and `Labelled` is British English.

The contributing docs don't specify, but I though we typically used American English spellings.

**Review by jeremylt — APPROVED** [review]
LGTM, modulo the Brittish vs American spelling issue.

**Review by Sembauke — APPROVED** [review]

**Comment by Sembauke** [comment]
Hi @tmsagarofficial,
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**